test/resources/secret-fixture.edn

```diff
@@ -0,0 +1,11 @@
+{:batman "Bruce Wayne"
```

Comment:
You should either put this file at the root of `test` or put it fully inside the `amperity/gocd/secret/vault/` path - there's nothing special about `test/resources`, so it's potentially confusing given that `resources` and `dev-resources` _are_ real top-level paths.